src/main/java/bio/terra/workspace/service/resource/controlled/ControlledResourceService.java

```diff
@@ -173,6 +175,17 @@ private JobBuilder commonCreationJobBuilder(
     return jobBuilder;
   }
 
+  private void validateFlightPrerequisites(
```

Comment:
nit: this is currently specific to `createControlledResource`, rename to `validateCreateFlightPrerequisites` or something similar?